cmd/promtool/main.go

```diff
@@ -131,8 +131,8 @@ func main() {
 	checkRulesCmd := checkCmd.Command("rules", "Check if the rule files are valid or not.")
 	ruleFiles := checkRulesCmd.Arg(
 		"rule-files",
-		"The rule files to check.",
-	).Required().ExistingFiles()
+		"The rule files to check, default is read from standard input (STDIN).",
```

Comment:
```suggestion
		"The rule files to check, default is read from standard input.",
```

I don't think it is stdin on all platforms (e.g. windows)